packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -936,14 +933,55 @@ export class SpecBuilder {
     }
   }
 
+  private addQueryToActiveTargetsIfNotYet(
```

Comment:
To be honest, I am not the biggest fan of this method name. Do you have alternatives?